deepchem/models/tests/test_layers.py

```diff
@@ -919,3 +919,47 @@ def test_torch_weighted_linear_combo():
     expected = torch.Tensor(input1) * layer.input_weights[0] + torch.Tensor(
         input2) * layer.input_weights[1]
     assert torch.allclose(result, expected)
+
+
+@pytest.mark.torch
+def test_edge_network():
+    """Test invoking the Torch equivalent of EdgeNetwork."""
+    n_pair_features = 2
+    n_hidden = 2
+    torch_init = 'xavier_uniform_'
+
+    # generate arbitrary features
+    pair_features = [[0.6445, 0.4471], [0.6969, 0.9996], [0.376, 0.9932],
+                     [0.2218, 0.3679]]
+    atom_features = [[0.1511, 0.8192], [0.1503, 0.3962], [0.9362, 0.9546],
+                     [0.6296, 0.954], [0.3062, 0.6088]]
+
+    atom_to_pair = []
+    n_atoms = 2
+    start = 0
+    C0, C1 = np.meshgrid(np.arange(n_atoms), np.arange(n_atoms))
+    atom_to_pair.append(
+        np.transpose(np.array([C1.flatten() + start,
+                               C0.flatten() + start])))
+
+    # tensors for torch layer
+    torch_pair_features = torch.Tensor(pair_features)
+    torch_atom_features = torch.Tensor(atom_features)
+    torch_atom_to_pair = torch.Tensor(atom_to_pair)
+    torch_atom_to_pair = torch.squeeze(torch_atom_to_pair.to(torch.int64),
+                                       dim=0)
+
+    torch_inputs = [
+        torch_pair_features, torch_atom_features, torch_atom_to_pair
+    ]
+
+    torch_layer = torch_layers.EdgeNetwork(n_pair_features, n_hidden,
+                                           torch_init)
+
+    # assigning tensorflow layer weights to torch layer
+    torch_layer.W = torch.from_numpy(np.load('assets/edgenetwork_weights.npy'))
```

Comment:
You've added 4 different .npy files but it looks like you're only using 2. Can you remove the unused .npy files?